src/pages/RealtimeMapPage.tsx

```diff
@@ -162,39 +156,25 @@ export default function RealtimeMapPage() {
         </Grid>
         <Grid xs={6}>
           <MinuteSelector
-            num={(+new Date(to) - +new Date(from)) / 1000 / 60}
-            setNum={(num) => setTo(formatTime(+new Date(from) + +num * 1000 * 60))}
+            num={to.diff(from)/1000/60}
+            setNum={(num) => {
+              setFrom(moment().subtract(num, 'minutes'))
+              setTo(moment())
+            }}
           />
         </Grid>
         <Grid xs={1}>
           <Label text={TEXTS.minutes} />
         </Grid>
         {/* Buttons */}
         {/*TODO (another PR another issue)
-          1) discussion if the buttons need to be 5 minutes from now or from the 'from' state.
+          1) discussion if the buttons need to be 5 minutes from now or from the 'from' state. 
+          => Done.
           2) recommended use MomentJs API instead Date API. 'moment().subtract(5, 'minutes')'.
-          3) use text `TEXTS`.
+          => Done.
+          3) use text `TEXTS`. => ?? were ?
```

Comment:
see this:
https://github.com/hasadna/open-bus-map-search/pull/95
we can do it on another PR